README.md

```diff
@@ -1925,6 +1927,25 @@ the list of flags you return from your `FlagsForFile` function in your
 
 See [issue #303][issue-303] for details.
 
+### Install YCM with [NeoBundle][NeoBundle]
+[NeoBundle][NeoBundle] can do the compilation for you just add in your vimrc:
+
+    NeoBundle 'Valloric/YouCompleteMe', {
+         \ 'build'      : {
+            \ 'mac'     : './install.sh --clang-completer --system-libclang --omnisharp-completer',
+            \ 'unix'    : './install.sh --clang-completer --system-libclang --omnisharp-completer',
+            \ 'windows' : './install.sh --clang-completer --system-libclang --omnisharp-completer',
+            \ 'cygwin'  : './install.sh --clang-completer --system-libclang --omnisharp-completer'
```

Comment:
On this part I think it would be better to to let like as suggested in the issue: https://github.com/Valloric/YouCompleteMe/issues/1082

This way even windows can try and give a feedback.